src/components/constants/Items.ts

```diff
@@ -408,6 +408,226 @@ export class Items {
         "Town Map": {
             "effect": "Shows your position in the Pokemon World",
             "category": "Key"
-        }
+        },
+        "HM01": {
+            "effect": "Cut",
```

Comment:
How do you plan this `effect` to map to the move it teaches? I would think you'd want it to be the same as the move name in some reliable way.